### DISCUSSION TITLE
add my own camera to the dataset

### DISCUSSION THREAD

**Discussion by joapatter** [openMVG/openMVG#2305] [Q&A]
hi! im having an issue with this example:

$ cd openMVG_Build/software/SfM/
$ python SfM_SequentialPipeline.py [full path image directory] [resulting directory]

when i use the example datasets (castle, dessert etc ...) everything is working fine.
The problem is when i use my own dataset, and i think the reason is that my camera model is not on the database, how can i add it????

1. Intrinsics analysis
INFO: [main_SfMInit_ImageListing.cpp:197]  You called : /home/jpaton/openMVG/openMVG_Build/Linux-x86_64-RELEASE/openMVG_main_SfMInit_ImageListing
--imageDirectory /home/jpaton/Escritorio/muestra1
--sensorWidthDatabase /home/jpaton/openMVG/src/software/SfM/../../openMVG/exif/sensor_width_database/sensor_width_camera_database.txt
--outputDirectory /home/jpaton/Escritorio/muestra1/matches
--focal -1
--intrinsics 
--camera_model 3
--group_camera_model 1
--use_pose_prior 0
--prior_weights 1.0;1.0;1.0
--gps_to_xyz_method 0
INFO: [loggerprogress.hpp:79] [- Listing images -] 100%
WARNING: [main_SfMInit_ImageListing.cpp:458] Warning & Error messages:
S1000332" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
S1000333" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
S1000334" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
S1000335" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
S1000336" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
S1000337" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
S1000338" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
S1000339" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
S1000340" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
S1000341" model "Skydio 2+" doesn't exist in the database
Please consider add your camera model and sensor width in the database.
metadata.csv: Unkown image file format.

**Comment by joapatter** [comment]
i solved it, go to openMVG/src/openMVG/exif/sensor_width_database.txt and simply add the model and the focal length

**Reply by pmoulon** [reply]
You did what was required. Sorry for not having being able to reply sooner. Happy SfM 😊
